ansible_collections/arista/avd/roles/eos_validate_state/tasks/reports.yml

```diff
@@ -9,8 +9,10 @@
   arista.avd.yaml_templates_to_facts:
     templates:
       - template: generate_state_report_results.j2
+      - template: generate_anta_report_results.j2
   delegate_to: localhost
   run_once: true
+  changed_when: false # yaml_template_to_facts always return changes in that case which break idempotency falsely
```

Comment:
Not sure why it always return changes? Maybe we are missing some sorting somewhere so it becomes random in order.

Reply:
It reports changed since we don't output a file. All ok.

Reply:
So as discussed because we don't have a destination, yaml_template_to_facts is (rightly) returning change but we can safely ignore it here as we are only generating some variable to register